File: go/10_input.go
package main

import "fmt"

func main() {
	var name string
	fmt.Print("What is your name? ")
	fmt.Scanln(&name)
	fmt.Printf("Hello %s!\n", name)

	var age int
	fmt.Print("How old are you? ")
	fmt.Scanln(&age)
	fmt.Printf("You are %d years old.\n", age)
	fmt.Printf("Next year you will be %d\n", age+1)
}

File: php/10_input.php
<?php
// 10 input

echo "What is your name? ";
$name = trim(fgets(STDIN));
echo "Hello $name!\n";

echo "How old are you? ";
$age = (int)trim(fgets(STDIN));
echo "You are $age years old.\n";
echo "Next year you will be " . ($age + 1) . "\n";
